Documentation/ceph-cluster-crd.md

```diff
@@ -646,3 +648,63 @@ spec:
           accessModes:
             - ReadWriteOnce
 ```
+
+### External cluster
+
+#### Pre-requisites
+
+In order to configure an external Ceph cluster with Rook, we need to inject some information in order to connect to that cluster.
+You can use the `cluster/examples/kubernetes/ceph/import-external-cluster.sh` script to achieve that.
```

Comment:
@leseb  - where can I find this script? I did not see it in https://github.com/rook/rook/tree/master/cluster/examples/kubernetes/ceph

Reply:
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/import-external-cluster.sh